pkg/run.go

```diff
@@ -145,6 +145,15 @@ func Run(c *cli.Context) (err error) {
 		}
 	}
 
+	vulnTypeSelector := c.String("vuln-type")
+	if vulnTypeSelector != "" {
+		utils.SetVulnTypeSelector(vulnTypeSelector)
+	} else {
+		utils.SetVulnTypeSelector("all")
+	}
+
+	log.Logger.Debugf("Vulnerability type:  %s", utils.VulnTypeSelector())
+
```

Comment:
Rather than relying on the global in utils, would it make sense to define a `ScanOptions` struct that could include the vuln types (for now, maybe more options later) and pass that as a parameter to `scanner.ScanImage` and `library.Scan`?

Reply:
I moved vulnType var and related functions in `pkg/types/scanoptions.go` file